src/moduler/aktivitet/visning/hjelpekomponenter/VarslingBoks.tsx

```diff
@@ -0,0 +1,52 @@
+import AlertStripe from 'nav-frontend-alertstriper';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { MOTE_TYPE } from '../../../../constant';
+import { Aktivitet } from '../../../../datatypes/aktivitetTypes';
+import { div as HiddenIfDiv } from '../../../../felles-komponenter/hidden-if/hidden-if';
+import Innholdslaster from '../../../../felles-komponenter/utils/Innholdslaster';
+import { selectDialogForAktivitetId, selectDialogStatus } from '../../../dialog/dialog-selector';
+import { selectErVeileder, selectIdentitetStatus } from '../../../identitet/identitet-selector';
+import { selectErBrukerManuell, selectReservasjonKRR } from '../../../oppfolging-status/oppfolging-selector';
+import { selectNivaa4 } from '../../../tilgang/tilgang-selector';
+
+interface Props {
+    className: string;
+    aktivitet: Aktivitet;
+}
+
+const VarslingBoks = ({ className, aktivitet }: Props) => {
+    const identitetStatus = useSelector(selectIdentitetStatus);
+    const dialogStatus = useSelector(selectDialogStatus);
+    const avhengigheter = [identitetStatus, dialogStatus];
+
+    const erVeileder = useSelector(selectErVeileder);
+    const dialogForAktivitetId = useSelector((state) => selectDialogForAktivitetId(state, aktivitet.id));
+    const erKRR = useSelector(selectReservasjonKRR);
+    const erManuell = useSelector(selectErBrukerManuell);
+    const harLoggetInnNivaa4 = useSelector(selectNivaa4);
+
+    const visVarselOmManglendeDialog =
+        aktivitet.type === MOTE_TYPE &&
+        erVeileder &&
+        !dialogForAktivitetId &&
+        !erKRR &&
+        !erManuell &&
+        harLoggetInnNivaa4;
```

Comment:
burde vi trekke ut dette i en hook?

Reply:
Er det dette som er trukket ut i en hook eller husker jeg feil? 🤔  Kan varsles virka litt kjent

Reply:
ja var det jeg syntes. føler vi implemeterer det på nytt over alt. Og da kommer vi til og glemme noe neste gang vi må nedre på det

Reply:
Aah useKanSendeVarsel blir ikke helt riktig 😢 

```
    const erManuell = useSelector(selectErBrukerManuell);
    const erKvp = useSelector(selectErUnderKvp);
    const erreservertKRR = useSelector(selectReservasjonKRR);
    const harNivaa4 = useSelector(selectNivaa4);

    return !(erManuell || erKvp || erreservertKRR || !harNivaa4);
```

Reply:
kanskje splitte den hooken i to? en som er erDigital og en som er kanSendevarsel?